apps/web/.env.example

```
@@ -119,6 +119,7 @@ NEXT_PUBLIC_AXIOM_TOKEN=
 
 # Transactional emails
 RESEND_API_KEY=
+NEXT_PUBLIC_IS_RESEND_CONFIGURED= # for frontend - enables relevant features
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)